ramls/feefinedata.json

```diff
@@ -37,6 +42,7 @@
   "additionalProperties": false,
   "required": [
     "feeFineType",
-    "ownerId"
+    "ownerId",
+    "automatic"
```

Comment:
I don't think FE adopted the use of this new flag yet, so manual fines are created without it at the moment. So maybe we shouldn't make it required now.